index.js

```diff
@@ -1,60 +1,67 @@
-/* eslint-disable no-use-before-define */
-const booksContainer = document.querySelector('.books');
-const titleInput = document.getElementById('title-input');
-const authorInput = document.getElementById('author-input');
-
-const addBtn = document.querySelector('.add-btn');
-
-let Books = JSON.parse(localStorage.getItem('books')) || [];
-
-addBtn.addEventListener('click', () => {
-  const titleValue = titleInput.value;
-  const authorValue = authorInput.value;
-  const bookID = Date.now();
-  addBook(titleValue, authorValue, bookID);
-});
-
-displayBooks();
-
-function addBook(title, author, id) {
-  const book = {
-    title,
-    author,
-    ID: id,
-  };
-  Books.push(book);
-  localStorage.setItem('books', JSON.stringify(Books));
-  displayBooks();
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Hi, you have done a great job, congratulations. Now, I would advise you not to disable Linter, you can solve the errors Manually.  Why is linter not allowed to be disabled:
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

There are a few reasons for using linters:

Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.